pyproject.toml

```
@@ -1,28 +1,25 @@
 [build-system]
 build-backend = "hatchling.build"
```

This line needs to be readded, otherwise this breaks the build
```suggestion
requires = ["hatchling"]
build-backend = "hatchling.build"
```